package.json

```diff
@@ -18,7 +18,7 @@
     "chai": "^3.5.0",
     "extend-me": "^2.3",
     "fin-hypergrid-data-source-base": "^0.4.10",
-    "finbars": "1.5.2",
+    "finbars": "github:dashdash/finbars#FEATURE/custom_scroll_speed",
```

Comment:
This needs to be updated after approving the [pull request](https://github.com/openfin/finbars/pull/2) in `finbars` repository.